README.md

```diff
@@ -19,13 +19,26 @@ By default, Tailscale release v1.20.3 is built. To configure the release number,
 set the environment variable `TSTAG` to the release number, e.g.
 `TSTAG=v1.20.3 make out/pkg`.
 
+How-To Build in Windows
+----------------
+1. Install "Windows Subsystem for Linux (WSL) https://docs.microsoft.com/en-us/windows/wsl/install"
+  1a. Quick Start WSL by entering this command in an administrator PowerShell or Windows Command Prompt and then restarting your machine `wsl --install`
+2. Install Windows Terminal https://aka.ms/terminal
+3. From Windows Command Prompt or PowerShell, you can open your default Linux distribution inside your current command line, by entering: `wsl.exe`
+4. When in WSL run `sudo apt install docker.io build-essential -y`
+5. `git clone https://github.com/ivokub/tailscale-qpkg.git`
+6. `sudo dockerd` (then open a new linux session to be able to run next line, if you ctrl-c the service stops)
+7. `cd tailscale-qpkg/`
+9. `sudo make out/pkg`
+10. You can find the files if you do "Win+R" command and paste `%USERPROFILE%\tailscale-qpkg\out\pkg`
+
 Installation
 ------------
 
 1. Manually install Tailscale package in QNAP App Center.
-2. SSH into your QNAP
+2. SSH into your QNAP and push Q in the menu to get shell.
```

Comment:
```suggestion
2. SSH into your QNAP. If instead of command line shell you see a list of options, choose Q to open shell.
```

Older versions of QNAP do not have the admin menu and it is possible to manually disable it. So, every user does not necessarily see the option.